src/data/guilty_gear_xrd_rev_2/jack-o.yml

```diff
@@ -0,0 +1,99 @@
+---
+name: Jack-O
+moves:
+  normals:
+    - 5P
+    - 5K
+    - c.S
+    - f.S
+    - 5H
+    - 6P
+    - 6K
+    - 6H
+    - 3H
+    - 5D
+    - name: Hunger of Dopoulos
+      inputs:
+        - 4D
+        - j.4D
+    - name: Elysion Driver
+      inputs:
+        - 6D
+        - j.6D
+    - 2P
+    - 2K
+    - 2S
+    - 2H
+    - name: Zest
+      inputs:
+        - 2D
+        - j.2D
```

Comment:
Is this redundant with Air Zest (j.2D) ?

Reply:
Buuuuuuh weird. Dustloop shows **air ok** on Zest, but also lists an Air Zest move.

Reply:
I'll remove the redudancy here though.